tools/build_defs/repo/jvm.bzl

```diff
@@ -146,12 +147,19 @@ def _concat_with_needed_slash(server_url, url_suffix):
     else:
         return server_url + "/" + url_suffix
 
+def _get_import_attribute(rule_name, attribute_value):
```

Comment:
@jin I understand the difficulty but it just feels that you get to do this change (couple this to aar) because you’re part of the team. If we were talking about supporting an external format explicitly would you still support it?
@dslomov, do you have a smart suggestion by any chance?

[@jin I’m not excusing but sharing how it looks from outside]

Reply:
I'm sorry you felt that way but that wasn't my intention. I wasn't comfortable with having an attribute value containing Python string formatting placeholders, which is why I hardcoded the if-else in the implementation.

PTAL at the new parameterized metadata attribute.

Reply:
To be super clear, I wasn’t implying ill  will but more of a blind spot.
Even with that I’m sorry if you felt I was implying incorrect intentions.